### PR TITLE
Add AI controlled maintenance drone

### PR DESCRIPTION
**Pull request by hyperioo**
## About The Pull Request

This PR adds the possibility for AI to print a maintenance drone that they can control remotely.

* Use `Control Personal Drone` verb in silicon commands to print a drone and control it.
* Use `Get Back To Core` verb to get back to your core when you are in the drone.

When you take control of the drone for the first time it prints the default drone laws but you actually have your AI laws (check in Show Laws). It's a default game behaviour that is a pain to remove.

This drone cannot be emmaged.

AI can only create a single drone every 30 minutes so you better take care of it!

Edge cases:
* AI out of power / AI is emp'ed / AI core dies / AI law are modified -> pulled back from drone into core
* drone out of power / drone dies / drone is emp'ed / drone goes out of ship zlevels -> automatically sent back to core
* intellicard used on AI -> pulled back into care then intellicard processes

Tell me if you think I forgot a edge case.

## Why It's Good For The Game

Possibility for AI to fix some things during low pops and/or when technomancers cannot connect their brain cells to fix the ship.

## Changelog
:cl: Hyperio
add: Add AI controlled maintenance drone
/:cl:

### CODE REVIEW

**Comment by Anonmare** [comment]
Does ID swiping insta-kill it still?

**Comment by hyperioo** [comment]
> 
> 
> Does ID swiping insta-kill it still?

No it does not.

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -731,3 +741,47 @@ var/list/ai_verbs_default = list(
...
+/mob/living/silicon/ai/proc/go_into_drone()
+	// Switch to drone or spawn a new one
+	if(!bound_drone)
+		testing("[world.time] . [time_destroyed] . [drone_cooldown_time]")
+		if (world.time - time_destroyed > drone_cooldown_time)
+			try_drone_spawn(src, aibound = TRUE)
+		else
+			var/remaining = (drone_cooldown_time - (world.time - time_destroyed)) / 10
+			to_chat(src, SPAN_WARNING("Security routines hardcoded into your core force you to wait [remaining] seconds before creating a new AI bound drone."))
+	else if(src?.mind)
```
Comment (TheShown911) [inline code/modules/mob/living/silicon/ai/ai.dm:781]: `src` should always exist, also you should be able to throw most of the `src.` out of the code

Comment (hyperioo) [reply code/modules/mob/living/silicon/ai/ai.dm:781]: True, I'll try to remove them 👍

**Comment by hyperioo** [comment]
Was tested last week and I did not hear any yell so I guess it's ok?

**Pull request merged**